assets/prefabs/vehicles/loco.prefab

```diff
@@ -0,0 +1,50 @@
+{
+    "Location": {
+        "scale": 0.45
+    },
+    "Mesh": {
+        "mesh": "AdvancedRails:loco",
+        "material": "AdvancedRails:loco"
+    },
+    "RigidBody": {
+        "mass": 500,
+        "collisionGroup": "engine:world",
+        "collidesWith": [
+            "engine:world",
+            "engine:debris",
+            "engine:character",
+            "engine:default"
+        ]
+    },
+    "HullShape": {
+         "sourceMesh" : "AdvancedRails:loco-bounds"
+    },
+    "CartJoint":{
+        "front": {
+            "range": 0.7
+        },
+        "back": {
+            "range": 0.7
+        }
+    },
+    "Health": {
+        "currentHealth": 1,
+        "maxHealth": 1,
+        "regenRate": 1,
+        "waitBeforeRegen": 2,
+        "destroyEntityOnNoHealth" : true
+    },
+    "Trigger": {
+        "detectGroups": [
+            "engine:world",
+            "engine:debris",
+            "engine:sensor",
+            "engine:kinematic",
+            "engine:default"
+        ]
+    },
+    "Rails:RailVehicle": {},
+    "AdvancedRails:Locomotive": {},
+    "Network": {
```

Comment:
You probably want to use `{}` here instead of using a two-liner? Probably both approaches are in use. Even if there isn't a consistent standard we should try to at least be consistent and pick one within a given file :-)